clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -77,6 +79,10 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
       if (!description.env) description.env = [:]
       if (!description.labels) description.labels = [:]
 
+      if (description.labels.containsKey(INTERESTING_HEALTH_PROVIDER_NAMES) && "Titus" in description.labels.get(INTERESTING_HEALTH_PROVIDER_NAMES)) {
```

Comment:
As currently coded, it's necessary. But will revisit the Orca side in the am. Might end up not needing the explicit set here afterall